community/planning/dlt_polling_monitor.md

```diff
@@ -67,6 +70,37 @@ The DLT Polling Monitor will work through the queue in the following manner:
    4. Updates db with `update_batch_statuses`
 4. Reruns on the specified `poll_interval`
 
+## Interaction with other services
+
+Below is a sequence diagram that outlines how the DLT polling monitor (Poller)
+interacts with the DLT Event Monitor, Grid Daemon, Ledger, and Database.
+
+<div class="mermaid">
+sequenceDiagram
+    participant Poller as DLT Polling Monitor
```

Comment:
This might be a little cleaner if Gridd came first

Reply:
Good point! Updated so that Gridd is the first item.